cockroachcloud/serverless-faqs.md

```diff
@@ -0,0 +1,95 @@
+---
+title: CockroachCloud Serverless (beta) FAQs
+summary: Get answers to frequently asked questions about CockroachCloud Serverless (beta)
+toc: true
+---
+
+This page answers the frequently asked questions about CockroachCloud Serverless (beta) and CockroachCloud Dedicated.
+
+<div class="filters clearfix">
+    <a href="serverless-faqs.html"><button class="filter-button page-level current">CockroachCloud Serverless</button></a>
+    <a href="frequently-asked-questions.html"><button class="filter-button page-level">CockroachCloud Dedicated</button></a>
+</div>
+
+## General
+
+### What is CockroachCloud Serverless (beta)?
+
+CockroachCloud Serverless (beta) delivers free CockroachDB clusters for you and your organization. It is a managed instance of CockroachDB that removes the friction of initial cluster sizing and auto-scales based on your application traffic.
```

Comment:
Are we set on talking about what this in negative terms? E.g. instead of "removes the friction of initial cluster sizing" we could say something like "lets you begin working with your database immediately."

Feel free to ignore this.